.circleci/config.yml

```diff
@@ -440,6 +451,40 @@ jobs:
           command: WP_MULTISITE=1 composer run test
           working_directory: /tmp/wordpress/wp-content/plugins/coblocks
 
+  unit_testing_php_74:
+    docker:
+      - image: circleci/php:7.4.15-apache-node
+        auth:
+          username: $DOCKERHUB_USERNAME
+          password: $DOCKERHUB_PASSWORD
+        environment:
+          XDEBUG_MODE=coverage
+      - image: circleci/mysql:5.7
+        auth:
+          username: $DOCKERHUB_USERNAME
+          password: $DOCKERHUB_PASSWORD
+    steps:
+      - checkout
+      - attach_workspace:
+          at: ~/project
+      - restore_composer_cache
+      - run:
+          name: Install Composer v1
+          command: |
+            sudo composer self-update --1
```

Comment:
@AnthonyLedesma I had mentioned this in slack, but just put this one onto a single line and then we can merge this.